pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -72,6 +67,12 @@ in buildPythonPackage rec {
   PYTORCH_BUILD_VERSION = version;
   PYTORCH_BUILD_NUMBER = 0;
 
+  USE_FBGEMM = 0; # this can't build because of CMAKE downloads
+  USE_MKLDNN = 0; # mkl with cuda is broken
+  USE_NCCL = 0; # multigpu looks broken broken
+  # on deck
+  USE_DISTRIBUTED = 0; # this might work
```

Comment:
Distributed fails because of the testing suite. I'll rebuild and pull out the stack trace.

Reply:
Seems to have resolved itself